test/e2e/framework/test_context.go

```diff
@@ -294,7 +299,7 @@ func RegisterClusterFlags() {
 	flag.StringVar(&cloudConfig.Cluster, "gke-cluster", "", "GKE name of cluster being used, if applicable")
 	flag.StringVar(&cloudConfig.NodeInstanceGroup, "node-instance-group", "", "Name of the managed instance group for nodes. Valid only for gce, gke or aws. If there is more than one group: comma separated list of groups.")
 	flag.StringVar(&cloudConfig.Network, "network", "e2e", "The cloud provider network for this e2e cluster.")
-	flag.IntVar(&cloudConfig.NumNodes, "num-nodes", -1, "Number of nodes in the cluster")
+	flag.IntVar(&cloudConfig.NumNodes, "num-nodes", DefaultNumNodes, "Number of nodes in the cluster. If the default value of '-1' is used the number of schedulable nodes is auto-detected.")
```

Comment:
if we have a constant this can now be:
```
... , "If the default value of %q is used the number of schedulable nodes is auto-detected.",  DefaultNumNodes)
```

Reply:
:+1: